server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsResponse.java

```diff
@@ -53,15 +48,15 @@ public GetSnapshotsResponse(
         final int remaining
     ) {
         this.snapshots = List.copyOf(snapshots);
-        this.failures = failures == null ? Map.of() : Map.copyOf(failures);
```

Comment:
Can we remove the constructor arg as well?

Reply:
Will do as soon we get TransportVersions.V9 :+1:

Reply:
Done!